src/plugin/vk-wrapper/imageView/ImageView.cpp

```diff
@@ -0,0 +1,46 @@
+/*
+** EPITECH PROJECT, 2024
+** Title: EngineSquared
+** Author: MasterLaplace
+** Created: 2024-10-23
+** File description:
+** ImageView
+*/
```

Comment:
why is the header different from GraphicsPipeline.hpp ?

Reply:
I've chosen two different header types for hpp and cpp. The first is more complete and can include doxygen documentation. The second is lighter and contains just simple information.